docs/src/conf.py

```diff
@@ -1,13 +1,25 @@
+import os
 from datetime import datetime
 
 
+ROOT = os.path.abspath(os.path.join("..", ".."))
+
 # Add any Sphinx extension module names here, as strings.
 extensions = [
     "sphinx.ext.viewcode",
     "sphinx.ext.intersphinx",
-    "sphinx_gallery.load_style",
+    "sphinx_gallery.gen_gallery",
```

Comment:
this should not be `gen_gallery`. We generate the galleries in a sperate step, this final sphinx build should just load pre-generated rst